build.gradle

```diff
@@ -50,3 +54,7 @@ wrapper {
   gradleVersion = "5.6.4"
   distributionType = Wrapper.DistributionType.BIN
 }
+
+compileGroovy.dependsOn.remove('compileJava')
```

Comment:
Why is this required?

Reply:
We need to call Groovy code from Kotlin, see this for more details:
https://github.com/gradle/gradle/issues/11333

Reply:
Ah thanks for the pointer. 👍